en-us/faq.md

```diff
@@ -0,0 +1,29 @@
+# FAQ
+
+### How to install?
+
+- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions) to install FO on the launcher that you use.
+
+### So I just copy the mods, right?
+
+- Well no because each FO instance / ZIP contains configs and a custom resource pack that is built to help the user navigate through Mod Menu. If you only copy the mods over, the configs or overrides would not apply and you would configure the mods yourself which could lead to not having the right settings to replicate OptiFine.
+
+### What is the MultiMC ZIP?
+
+- MultiMC ZIP is packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
+
+### What is in the ZIP file?
+
+- In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. Without these files, the modpack would not be configured properly which could lead to not having the right settings to replicate OptiFine.
```

Comment:
this should be an hierarchical overview of each folder and important file, however you do not have to mention individual config files (so just combine them under "mod configs" or so)

example:

* folder1 - description
* folder2 - description
   * subfolder1 - description